### PR TITLE
SAML role and team sync (open source part)

### PR DESCRIPTION
**Pull request by alexanderzobnin**
**What this PR does / why we need it**:
This is an Open Source part of SAML role and team sync.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by xlson** [comment]
Instead of adding more options to the OSS config I think we should move all SAML config to Enterprise instead.

**Comment by alexanderzobnin** [comment]
@xlson what do you mean? Create separate `.ini` file, like `enterprise.ini`?

**Comment by aknuds1** [comment]
Sounds like a good idea @xlson.

**Comment by xlson** [comment]
@alexanderzobnin No, we don't need to create a new ini file. But we can migrate the inclusion of these configuration options into enterprise instead of in grafana. I'll send you a link on slack.

File: pkg/api/login.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func (hs *HTTPServer) validateRedirectTo(redirectTo string) error {
 	return nil
 }
 
-func (hs *HTTPServer) cookieOptionsFromCfg() middleware.CookieOptions {
+func (hs *HTTPServer) CookieOptionsFromCfg() middleware.CookieOptions {
```
Comment (xlson) [inline pkg/api/login.go:48]: Do we need the cookies options to be public?

**Review by xlson — APPROVED** [review]
Great getting the SAML config moved to enterprise.

File: pkg/api/login.go (hunk 1 above)
Comment (alexanderzobnin) [reply pkg/api/login.go:48]: Yes, I use it in SAML PostACS() handler to make redirect after login working properly.

Comment (alexanderzobnin) [reply pkg/api/login.go:48]: https://github.com/grafana/grafana-enterprise/blob/a2168a13cf492de8b6a9b2897e501cc68917e5d7/src/pkg/extensions/saml/api.go#L172-L175

Comment (xlson) [reply pkg/api/login.go:48]: Ah.

**Pull request merged**